components/inputs/test/input-time.test.js

```diff
@@ -102,14 +106,30 @@ describe('d2l-input-time', () => {
 
 		it('should not save input seconds after time changes', async() => {
 			//Seconds are saved when value is assigned directly, not when input by user (for EOD)
-			const elem = await fixture('<d2l-input-time value="11:22:33"></d2l-input-time>');
+			const elem = await fixture(fixtureWithValue);
 			expect(elem.getTime().second).to.equal(33);
 			getInput(elem).value = '11:45 AM';
 			dispatchEvent(elem, 'change', false);
 			await oneEvent(elem, 'change');
 			expect(elem.value).to.equal('11:45:00');
 		});
 
+		it('should update value when dropdown changes', async() => {
+			const elem = await fixture(fixtureWithValue);
+			expect(elem.getTime(elem).second).to.equal(33);
```

Comment:
I don't typically like asserting 2 things in a single test. Instead, I like building up my tests such that the other tests have already covered certain assumptions. i.e. have a test that asserts that when you give it a value, that the value is correctly set. Then you don't need to worry about asserting that anywhere else, and this test can just test the one thing it wants to -- that when the dropdown is clicked, the value is reflected.